### DISCUSSION TITLE
May I ask why the hop-revived.nvim plugin was deleted?

### DISCUSSION THREAD

**Discussion by mkdir700** [FluxxField/bionic-reading.nvim#62] [General]
I saw this plugin on Reddit and am currently using it, but the repository for this plugin has been deleted. May I ask why? Thank you.

**Comment by FluxxField** [comment]
I made a mistake when I uploaded my version that I have locally. I was lazy and just copied my files over and pushed it when I should of forked it. Which was an unintentional misrepresentation of the code. So, to avoid any confusion and conveying the idea that the code was originally mine i simply removed it.

I just simply need to fork the original repo (which I should have in the first place) and push my fix to it.

But, if at the moment you need anything from it please let me know and I will be happy to help if I can

**Reply by mkdir700** [reply]
Thank you for your reply. I still have the source code of hop-revived.nvim locally, so I currently have no issues.

I hope you can restore it. Thanks again.

**Reply by FluxxField** [reply]
I know it's been a very long time. But, I have created a new plugin called smart-motion.nvim as my hop/flash/leap/sneak replacement
